libs/community/langchain_community/llms/huggingface_hub.py

```diff
@@ -127,7 +127,7 @@ def _call(
                 response = hf("Tell me a joke.")
         """
         _model_kwargs = self.model_kwargs or {}
-        parameters = {**_model_kwargs, **kwargs}
+        parameters = {"return_full_text": False**_model_kwargs, **kwargs, "return_full_text": False}
```

Comment:
Making it the first parameter looks better to me. Please notice that you have one syntax error and a typo in your last commit: you are missing a comma after the first item of the dictionary and you have `return_full_text` twice.